### ISSUE TITLE
Unable to use fresh install from Sylius Standard 1.8

### ISSUE THREAD

**Issue by macintoshplus** [Sylius/Sylius-Standard#429]
After create projet with composer, this error is throw:

```
In getSecurityCheckerCommandService.php line 9:
                                                                                                  
  Attempted to load class "SecurityCheckerCommand" from namespace "SensioLabs\Security\Command".  
  Did you forget a "use" statement for another namespace?
```

**Reproduce:**

```shell
composer create-project sylius/sylius-standard test1.8 1.8
cd test1.8
bin/console
```

**Solution 1**

Delete files:

* config/package/dev/security_checker.yaml
* config/package/test/security_checker.yaml
* config/package/test_cached/security_checker.yaml

**Solution 2**

Execute this command: `composer require --dev sensiolabs/security-checker`

This command restore the dependency lost between sylius-standard 1.7 and 1.8

**Comment by lchrusciel** [comment]
Hey @Jean-Baptiste,

Thanks for reporting. This issue should be solved with: #427

**Comment by lchrusciel** [comment]
For anyone else looking for providing such a solution in their application, we recommend the usage of `symfony` binary or including it in your project explicitly with `composer require --dev sensiolabs/security-checker`
